modules/sample-modules/extend-product-page.md

```diff
@@ -515,7 +514,7 @@ Finally, add this `CustomTabType` to the `$productFormBuilder` variable in your
 
 ```php
 /**
- * @param int|null $productId
+ * @param int $productId
```

Comment:
there's something fishy in this example, `$productId` is casted as an integer, but in the function it's used as a `ProductId` value object:

```php
$idValue = $productId ? $productId->getValue() : null;
````